docs/kubernetes/ingress-usage.md

```diff
@@ -240,6 +240,26 @@ a unique value. Read more about this in our
 
      flowId("reuse")
 
+# Filters - return fast
+
+Sometimes you just want to return a header, redirect or even static
+html content. You can return from skipper without doing a proxy call
+to a backend, if you end your filter chain with `<shunt>`.
+
+## Return static content
+
+To set a response header `X: bar`, return
+`<html><body>hello</body></html>` in the HTTP response body and return
+HTTP status code 200 from ingress itself, you can use the following
+filter chain:
+
+    zalando.org/skipper-filter: |
+      setResponseHeader("X", "bar") -> inlineContent("<html><body>hello</body></html>") -> status(200) -> <shunt>
+
+Keep in mind you need a valid backend definition to backends which are
+available, otherwise Skipper would not populate the route for safety
+reasons.
```

Comment:
I'd simplify the statement to:
"Keep in mind that you need a valid backend definition otherwise Skipper will not accept the entire route definition"

Is it a "safety"/security concern or just plain syntax failure?

Reply:
1. It's not security, but safety
2. if you have no backend this might be a syntax error
3. if you have a backend definition without having a kubernetes service for that, skipper will not create a route for safety reasons.